### ISSUE TITLE
Shot Generator doesn't load

### ISSUE THREAD

**Issue by LuckyLewis** [wonderunit/storyboarder#2408]
The _Shot Generator_ doesn't show up anymore. What I get instead is just an empty window like this one:

![imagen](https://user-images.githubusercontent.com/49131630/211637526-e31e2fd4-9009-465e-bbea-08f537460e0c.png)

I'm using desktop _Win10_ but the strange thing is that I was working with the _Shot Generator_ for almost two years with no trouble at all.

I've checked the log to track the issue:
```
[2023-01-10 19:49:28.349] [info] ready!
[2023-01-10 19:49:28.412] [error] UnhandledRejection SyntaxError: Unexpected token   in JSON at position 0
    at JSON.parse (<anonymous>)
    at e.exports._loadFile (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:327:145203)
    at new e.exports (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:327:145062)
    at C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:630:148497
    at Object.useMemo (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:359:56655)
    at useMemo (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:335:5275)
    at C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:630:148446
    at Xi (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:359:53107)
    at La (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:359:59921)
    at Sa (C:\Program Files\Storyboarder\resources\app.asar\src\build\shot-generator.js:359:59740)
```

I'd say that I haven't installed anything since the last time it worked fine, so I have no clue at all about where is this coming from.

**Comment by LuckyLewis** [comment]
Well, I have an idea.

It may be a problem with the name of some object created and saved as template in the _Shot Generator_. I'm from Spain and I've could used an extravagant char or something due to a different charset.

I'm gonna look for the path in which the created objects are stored.

**Comment by LuckyLewis** [comment]
Solved!

Nothing related to what I've said.

Somehow, it seems like the file _shot-generator-settings.json_ had been corrupted. I've changed it's name for I try and the next time I've opened the _Shot Generator_ it's generated a new settings file.

Now, it works fine again.

For the record, this is the location of the file that was causing the issue:
`C:\Users\[MyUser]\AppData\Roaming\Storyboarder\shot-generator-settings.json`
